src/cunumeric.mk

```diff
@@ -50,6 +50,7 @@ GEN_CPU_SRC += cunumeric/ternary/where.cc               \
 							 cunumeric/matrix/trsm.cc                 \
 							 cunumeric/matrix/util.cc                 \
 							 cunumeric/random/rand.cc                 \
+							 cunumeric/search/argwhere.cc              \
```

Comment:
a nitpick: any reason that you don't align the backslash with the rest?

Reply:
Nope, just an accident. I will adjust the alignment.